07.ls_object/lib/screen.rb

```diff
@@ -4,12 +4,16 @@
 require_relative 'detail_formatter'
 
 class Screen
-  def initialize(stats_in_dir, options = {})
-    @stats_in_dir = stats_in_dir
+  def self.show(ls_file_stats, options = {}, header: false)
+    new(ls_file_stats, options, header:).show
+  end
+
+  def initialize(ls_file_stats, options = {}, header: false)
+    @stats = ls_file_stats
```

Comment:
寿命の長い変数なので

```suggestion
    @ls_file_stats = ls_file_stats
```

とした方がLsFileStatの配列だということが伝わりやすくて嬉しいかも